Features/Context/SubContext/CommonActions.php

```diff
@@ -226,8 +226,17 @@ public function clickActionBar($button)
      */
     public function clickEditActionBar($button)
     {
-        $this->clickElementByText($button, '.ez-editactionbar-container .ez-action', '.action-label');
-        $this->waitWhileLoading();
+        $actionBarElements = $this->findAllWithWait('.ez-editactionbar-container .ez-action .action-label');
```

Comment:
This change has been made because in some cases the previous approach was not efficient:
If elements have been found using the main selector and weren't visible, the spin function waited the whole timeout for them to become visible (and only then processing the next item in array). This resulted in a huge execution time of this step (clicking the button took ca. 60 seconds, now it's ca. 3 seconds).